src/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/TciOnFdGrid.cpp

```diff
@@ -75,27 +74,19 @@ std::tuple<int, evolution::dg::subcell::RdmpTciData> TciOnFdGrid::apply(
       subcell_mesh.extents(),
       evolution::dg::subcell::fd::ReconstructionMethod::DimByDim);
 
-  Scalar<DataVector> dg_sqrt_det_spatial_metric{};
-  assign_data(make_not_null(&dg_sqrt_det_spatial_metric), num_dg_pts);
-  evolution::dg::subcell::fd::reconstruct(
-      make_not_null(&get(dg_sqrt_det_spatial_metric)),
-      get(sqrt_det_spatial_metric), dg_mesh, subcell_mesh.extents(),
-      evolution::dg::subcell::fd::ReconstructionMethod::DimByDim);
-
-  if (vars_needed_fixing and
-      (max(get(dg_tilde_d) / get(dg_sqrt_det_spatial_metric)) >
-       tci_options.atmosphere_density) and
-      (max(get(subcell_tilde_d) / get(sqrt_det_spatial_metric)) >
-       tci_options.atmosphere_density)) {
-    return {+1, rdmp_tci_data};
-  }
-
   if (min(get(dg_tilde_d)) <
           tci_options.minimum_rest_mass_density_times_lorentz_factor or
       min(get(dg_tilde_ye)) <
           tci_options.minimum_rest_mass_density_times_lorentz_factor *
               tci_options.minimum_ye or
       min(get(dg_tilde_tau)) < tci_options.minimum_tilde_tau) {
+    return {+1, rdmp_tci_data};
+  }
+
+  const bool not_in_atmosphere =
```

Comment:
In general it's better to have bools be positive. That is, `in_atmosphere` is more intuitive in terms of a bool than `not_in_atmosphere` because when the bool is `true` then we are not in the atmosphere... SpEC has some places where this is like 3 levels of `not`s and it's impenetrable. This is a general C++/software guideline :)

Reply:
Okay!